app/src/components/widgets/charts/common/utils/utils.js

```diff
@@ -52,6 +52,11 @@ const range = (start = 0, end = 0, step = 1) => {
 
 export const getItemNameConfig = (name) => {
   const nameParts = name.split('$');
+
+  if (nameParts.length === 1) {
+    return null;
```

Comment:
Maybe it's better to return an empty object? I see many places where this result is used without any checks for `null`.

Reply:
an empty object would be a truthy value, there are many checks as boolean

Reply:
Example: app/src/components/widgets/charts/launchStatisticsChart/launchStatisticsChart.jsx#323
```
const keyConfig = getItemNameConfig(key);
chartData[key] = [key];
colors[key] = getItemColor(keyConfig, defectTypes);
```
Where:
```
export const getItemColor = ({ itemType, defectType, locator }, defectTypes) => {
```
So it will lead to attempting do destructure `null`.

And there are many examples of such usages.

Reply:
fixed